_episodes/09-pull-requests.md

```diff
@@ -192,99 +291,224 @@ Largest City: Paris
 {: .output}
 
 Next we will add and commit these changes.
-Then we will push them to our forked version of the repo.
+Then we can push them to our forked version of the repo.
 
 ~~~
 $ git add france.txt
 $ git commit -m "Added largest city to france file"
-$ git push origin master
+$ git push origin addFrance
 ~~~
 {: .bash}
 
+~~~
+Counting objects: 3, done.
+Delta compression using up to 4 threads.
+Compressing objects: 100% (3/3), done.
+Writing objects: 100% (3/3), 387 bytes | 0 bytes/s, done.
+Total 3 (delta 0), reused 0 (delta 0)
+To https://github.com/sstevens2/countries.git
+   31aa2e3..609acfe  addFrance -> addFrance
+~~~
+{: .output}
+
 Now if we reload the pull request.  The new commit was added to that pull request.
+New commits pushed to the same branch are included in the previous pull request.
+If you want to suggest changes separately you need to make separate branches but 
+if you want the changes to be considered together you should put them in the same branch.
+
+![](../fig/github_screenshot_after_new_commit.png)
+
+When working with others we might encounter the conflicts, which we
```

Comment:
Yeah I've been conflicted about this as well.  I was thinking about adding this as an exercise but it is something you have to do with the instructor (since it requires two people making the same changes).  I kind of see it more as an optional instructor lead exercise?  So if you don't get to it it is okay but if you have time, I think it would be fun.

I think I'm going to leave it for now but we could discuss more about if we should cut it out or rearrange it or something after the workshop.